src/manager/index.ts

```diff
@@ -379,7 +440,8 @@ export default class Manager implements IManager, vscode.Disposable {
 
     this.updateChannelsFetchedAt();
     this.viewsManager.updateTreeViews();
-    this.fetchUnreadCounts(channels);
+    // TODO: re-enable fetch unread counts
+    // this.fetchUnreadCounts(channels);
```

Comment:
does this need to be commented? figure out why ids for groups are returning `channel_not_found`

reference: https://api.slack.com/changelog/2018-09-more-reasons-to-be-a-conversations-api-convert